libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -60,6 +60,7 @@ import { DhChooseOrganizationStepComponent } from './steps/dh-choose-organizatio
 import { DhNewOrganizationStepComponent } from './steps/dh-new-organization-step.component';
 import { DhNewActorStepComponent } from './steps/dh-new-actor-step.component';
 import { ActorForm } from './dh-actor-form.model';
+import { dhNameMaxLengthValidatorFn } from '../dh-name-max-length.validator';
```

Comment:
Question: Should this be used on all names or just actor names? If so maybe the name should dhActornameMaxLengthValidatorFn?

Reply:
Just the actor/market participant name. Just pushed a fix :)